File: backend/Models/Challenges/Challenge.cs
namespace Models.Challenges;

public class Challenge : EntityInfo, IEntity
{
    public string Title { get; set; } = null!;
    public string? ShortDescription { get; set; }
    public string Description { get; set; } = null!;
    public Guid CategoryId { get; set; }
    public bool IsCatInBag { get; set; }
}

File: frontend/src/types/challenge.ts
export type Challenge = {
  id: string;
  categoryId: string;
  title: string;
  shortDescription?: string;
  description?: string;
  isCatInBag: boolean;
  order?: number;
};

export type CreateChallenge = Omit<Challenge, "id" | "order">;
